docs/06-contributors/999-rfcs/2023-04-16-website-resource.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Website resource
+description: How to implement the website resource
+---
+
+# Website resource
+
+- **Author(s):**: @tsuf239
+- **Submission Date**: 2023-04-16
+- **Stage**: draf
+- **Stage Date**: 2023-04-16
+
+<!---
+Directions for above:
+
+Author(s): Git tag for PR authors and contributors
+Submission Date: Fill in with date of initial submission, YYYY-MM-DD
+Stage: Fill in with current stage in the RFC lifecycle
+Stage Date: Fill in with date of last stage change
+-->
+
+> Implementing the website resource following the [SDK spec](../../04-reference/wingsdk-spec.md#website).
+
+<!--
+This RFCs have 3 sections: Requirements -> Design -> Implementation. We intentionally start with *Design* since it
```

Comment:
I'd expect to see these 3 sections. If the SDK spec contains most of the Requirements and Design then I would at least say so. It does feel like this RFC jumps too quicly into the details without providing any introduction

Reply:
The requirements are the "contract" of the feature you're developing - "what does it do?" (as opposed to "how does it do it" - the implementation). The requirements usually specify use cases as well as edge case scenarios and the desired behavior of the software described.
Practically speaking, you can take a look at other resources in the SDK spec - some good examples are the first paragraphs of the [Function resource](https://docs.winglang.io/reference/wingsdk-spec#function) and the [Queue resource](https://docs.winglang.io/reference/wingsdk-spec#queue).

Reply:
cool, I added your explanation to the template :)